packages/cashier/src/pages/account-transfer/account-transfer.tsx

```diff
@@ -91,9 +87,6 @@ const AccountTransfer = observer(({ onClickDeposit, onClickNotes, onClose, setSi
     if (is_transfer_confirm) {
         return <AccountTransferReceipt onClose={onClose} />;
     }
-    if (!is_from_outside_cashier && is_crypto_transactions_visible) {
-        return <CryptoTransactionsHistory />;
-    }
```

Comment:
We never render the `CryptoTransactionsHistory />` inside the `<AccountTransfer />` page, hence removing it.